src/ui/constants/common.ts

```diff
@@ -287,7 +286,7 @@ export enum Scope {
 }
 
 // TODO: delete after update
-export const ALL_SCOPES: string[] = [...Object.values(Scope), EntryScope.Report];
+export const ALL_SCOPES: string[] = [...Object.values(Scope)];
```

Comment:
the array spread is not necessary
`export const ALL_SCOPES: string[] = Object.values(Scope);`

And perhaps you can delete TODO?